MaterialLib/SolidModels/CreepBGRa.cpp

```diff
@@ -77,14 +89,13 @@ CreepBGRa<DisplacementDim>::integrateStress(
         // side effect
         s_n1 = deviatoric_matrix * solution;
         double const norm_s_n1 = Invariants::FrobeniusNorm(s_n1);
+        double const G2b = 2.0 * b * this->_mp.mu(t, x);
         // side effect
-        pow_norm_s_n1_n_minus_one_2b_G =
-            2.0 * b * this->_mp.mu(t, x) * std::pow(norm_s_n1, n - 1);
-        jacobian =
-            KelvinMatrix::Identity() +
-            pow_norm_s_n1_n_minus_one_2b_G *
-                (deviatoric_matrix +
-                 ((n - 1) / (norm_s_n1 * norm_s_n1)) * s_n1 * s_n1.transpose());
+        pow_norm_s_n1_n_minus_one_2b_G = G2b * std::pow(norm_s_n1, n - 1);
+        jacobian = KelvinMatrix::Identity() +
+                   (pow_norm_s_n1_n_minus_one_2b_G * deviatoric_matrix +
+                    (n - 1) * G2b * std::pow(norm_s_n1, n - 3) * s_n1 *
```

Comment:
Replaced `pow_norm_s_n1_n_minus_one_2b_G/ (norm_s_n1 * norm_s_n1)` with `G2b * std::pow(norm_s_n1, n - 3)`  in order to  avoid `[...]/|s|^{2}`. Normallly, `n` is a real number around 5.0